mapswipe_workers/mapswipe_workers/definitions.py

```diff
@@ -1,34 +1,58 @@
-import json
 import logging
 import logging.config
 import os
 
 import sentry_sdk
-from xdg import XDG_CONFIG_HOME, XDG_DATA_HOME
+from xdg import XDG_DATA_HOME
+
+from mapswipe_workers.config import SENTRY_DSN
 
 
 class CustomError(Exception):
     pass
 
 
-def load_config(CONFIG_PATH) -> dict:
-    """Read the configuration file."""
-    with open(CONFIG_PATH) as f:
-        return json.load(f)
-
-
-CONFIG_DIR = os.path.join(XDG_CONFIG_HOME, "mapswipe_workers")
-
-CONFIG_PATH = os.path.join(CONFIG_DIR, "configuration.json")
-
-CONFIG = load_config(CONFIG_PATH)
-
-SERVICE_ACCOUNT_KEY_PATH = os.path.join(CONFIG_DIR, "serviceAccountKey.json")
+IMAGE_URL = {
```

Comment:
just to double check:
is this getting us the right strings we need later for project creation?

Reply:
in auth.py we use the `get_tileserver_function`

```
IMAGE_URL['bing']
('https://ecn.t0.tiles.virtualearth.net', '/tiles/a{quad_key}.jpeg?g=7505&mkt=en-US&token={key}')
```

Reply:
No, you are right. This will return a tuple instead of a string. Corrected `("...", "...")` to `("..."+"...")`. Thanks.